ci-conda.Dockerfile

```diff
@@ -110,14 +109,31 @@ rapids-mamba-retry install -y \
   anaconda-client \
   boa \
   gettext \
-  gh \
   git \
   jq \
-  "sccache==${SCCACHE_VER}" \
   "python=${PYTHON_VERSION}.*=*_cpython"
 conda clean -aipty
 EOF
 
+# Install sccache and gh cli
+ARG SCCACHE_VER
+ARG REAL_ARCH
+ARG GH_CLI_VER=notset
+ARG CPU_ARCH
+RUN <<EOF
+set -e
```

Comment:
```suggestion
```

This line shouldn't be needed because of the `SHELL` directive earlier in this file.